ckan/logic/action/create.py

```diff
@@ -27,12 +25,10 @@
 import ckan.lib.datapreview
 import ckan.lib.api_token as api_token
 import ckan.authz as authz
-
-from ckan.common import _, config
-
-# FIXME this looks nasty and should be shared better
```

Comment:
I agree with the comment here about the import of `_update_package_relationship`, why are you removing it?